### PR TITLE
added mariadb so that demo could use mysql browser to show the result…

### PR DESCRIPTION
**Pull request by wxlund**
Added mariadb driver so that demo could use mysql browser to show the results in the meta tables and person table for better demo.  Also added @enableTask to get the expected result in task execution list

dataflow:>task execution list
╔══════════╤══╤════════════════════════════╤════════════════════════════╤═════════╗
║Task Name │ID│         Start Time         │          End Time          │Exit Code║
╠══════════╪══╪════════════════════════════╪════════════════════════════╪═════════╣
║fileIngest│3 │Thu Apr 04 11:07:56 PDT 2019│Thu Apr 04 11:08:00 PDT 2019│0        ║
║fileIngest│2 │                            │                            │         ║
╚══════════╧══╧════════════════════════════╧════════════════════════════╧═════════╝

#2 was without the @enableTask
#3 was with @enableTask

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@wxlund Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-dataflow-samples&pullRequestId=94)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-dataflow-samples&pullRequestId=94) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@wxlund Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-dataflow-samples&pullRequestId=94)!

File: batch/file-ingest/pom.xml [hunk 1]
```diff
@@ -1,155 +1,163 @@
...
-    <reporting>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-checkstyle-plugin</artifactId>
-                <version>${checkstyle.plugin.version}</version>
-            </plugin>
-        </plugins>
-  </reporting>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (chrisjs) [inline batch/file-ingest/pom.xml]: it looks like there are changes intermingled with an entire file re-format which makes it hard to fish everything out.. re-formatting is fine, but can you do that in another commit so the non-reformatting changes are clear?

File: batch/file-ingest/pom.xml [hunk 2]
```diff
@@ -1,155 +1,163 @@
...
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-batch</artifactId>
+		</dependency>
+		<!-- https://mvnrepository.com/artifact/org.mariadb.jdbc/mariadb-java-client -->
+		<dependency>
+			<groupId>org.mariadb.jdbc</groupId>
+			<artifactId>mariadb-java-client</artifactId>
+			<version>2.4.1</version>
```
Comment (chrisjs) [inline batch/file-ingest/pom.xml]: theres a specific block below - the `kubernetes` `profile` that specifically has the mariadb driver. these two blocks differ in versions as well.

have these changes been tested on all supported platforms per the sample ref guide? if its applicable to all platforms, then id guess it could be removed from the specific profile below

File: batch/file-ingest/src/main/resources/application.properties [hunk 3]
```diff
@@ -1,2 +1,6 @@
 spring.application.name=fileIngest
 spring.datasource.initialization-mode=always
+spring.datasource.url=jdbc:mysql://mysql:3306/dataflow
```
Comment (chrisjs) [inline batch/file-ingest/src/main/resources/application.properties]: are these values actually needed in the properties file or might this be left over from debugging? in for example k8s or docker compose, these values are passed already via vars in their respective config files. id assume on CF there may be some auto-configuration or whatnot

since we didn't have this before, can you verify this is actually needed now across the supported platforms?

**Comment by wxlund** [comment]
I restored the pom.xml and application.properties to their original so that the pull request is only to add the @EnableTask.  I don’t have the envs or bandwidth to test on all platforms. Let me know if this resolves the issues you listed.  These properties were in the original:

> spring.application.name=fileIngest
> spring.datasource.initialization-mode=always
And I left them untouched and removed the mysql specific properties.

**Comment by chrisjs** [comment]
merged in bcd56edaa19b272

the file-ingest sample gets built upon by other samples, so i think by just adding back `@EnableTask` here was the proper thing to do. as with the other samples that get built on top, requirements start changing based on the sample. in the compose situation such as yours, things start to change as things aren't truly "local" anymore -- mysql is needed to record data as things are separated via containers, the maria driver is needed to make that connection, etc. if it makes sense, might make for an additional sample document.
